cli/tests/test/clear_timeout.ts

```diff
@@ -0,0 +1,5 @@
+clearTimeout(setTimeout(() => {}, 1000));
```

Comment:
I don't think this would ever trigger sanitizers - timeout is cleared (and thus op is done) before the first test runs

Reply:
Yeah I'm not sure why it did (internally async maybe?), but apparently it did a few months ago in #7844.